configure.ac

```diff
@@ -2927,7 +2943,7 @@ esac
 AC_MSG_CHECKING(for --with-address-sanitizer)
 AC_ARG_WITH(address_sanitizer,
             AS_HELP_STRING([--with-address-sanitizer],
-                           [enable AddressSanitizer (asan)]),
+                           [enable AddressSanitizer memory allocation controller, 'asan' (default is no)]),
```

Comment:
If 'AddressSanitizer' must be described, call it 'memory error detector' instead. I would rather leave it as-is; it's a specific thing, and an easily googleable name.